components/Typography/tests/unit/UIFont+MaterialTypographyTests.m

```diff
@@ -69,7 +69,7 @@ - (void)testScaledFontForTraitEnvironmentWithNoApplicationOniOS9DoesNothing {
   }
 
   // Given
-  XCTAssertNil(UIApplication.sharedApplication);
+  // XCTAssertNil(UIApplication.sharedApplication);
```

Comment:
It looks like this was accidentally left commented-out, since Typography isn't affected by Ripple. Please uncomment this line.